### ISSUE TITLE
Windows: Unexpected titlebar for frameless and transparent windows

### ISSUE THREAD

**Issue by felixrieseberg** [electron/electron#11849]
We're seeing a bug where `BrowserWindows` created with `transparent: true, frame: false` show a title bar in various states of user interaction.

* Electron version: 1.8.2-beta.5 (not present on beta.4, like [due to this change](https://github.com/electron/electron/pull/11710))
* Operating system: Windows

### Expected behavior

`BrowserWindows` created with `transparent: true, frame: false` should never show a title bar.

### Actual behavior

`BrowserWindows` created with `transparent: true, frame: false` show a title bar in various states of user interaction.

![frameless](https://user-images.githubusercontent.com/1426799/35893110-5588740c-0b61-11e8-8d10-73838a1f88d7.JPG)

### How to reproduce

Run the following:

```sh
git clone https://github.com/felixrieseberg/electron-quick-start
git checkout frame-titlebar-issue
npm start
```

Then, select the window in the taskbar, or click on it. The repro steps vary a bit, but you should eventually see a title bar.

**Comment by bbatliner** [comment]
![image](https://user-images.githubusercontent.com/5896658/36078582-357dcb5e-0f3e-11e8-8333-14b5c54b02c2.png)

I'm also experiencing this issue on Windows 10.

Notably, the bar is not present when I first launch my app. Only after Alt+Tabbing to a new window or clicking out of Electron does the title bar appear.

**Comment by brenca** [comment]
@GryoMx The transparency being lost after the devtools is closed is a different bug, I proposed a fix in #11233 but that was rejected. I couldn't find the root cause of the problem, I might take a shot at it again.

**Comment by brenca** [comment]
The title bar thing only happens on 1.8.2-beta.5 and 1.8.2, master isn't affected apparently. I'm going to post a fix in a minute, it's basically just re-enabling the `WS_EX_COMPOSITED` style when GPU acceleration is on.

**Comment by alexeykuzmin** [comment]
@brenca 
Since the PR above is merged, can this issue be closed?

**Comment by alexeykuzmin** [comment]
> @brenca
Since the PR above is merged, can this issue be closed?

up

**Comment by alexeykuzmin** [comment]
It seems to be fixed in 1.8.3 by #11910. Closed.
